channel_check.py

```
@@ -16,7 +16,7 @@ def open_json():
     # open configuration file
     os.chdir("./")
     with open(
-        f"{os.getcwd()}/DataBase/server.config.json", "r", encoding="utf-8"
+        f"{os.getcwd()}/database/server.config.json", "r", encoding="utf-8"
```

Ruff (0.14.7)

Unnecessary mode argument

Remove mode argument

(UP015)